mitodl/open-discussions
# Repair posts if they don't appear in the hot posts list

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1955 

#### What's this PR do?
Repairs posts that don't show up in the hot listing by having the indexing user upvote then clear the vote.

#### How should this be manually tested?
Create a post, ensure you don't get any errors. It's impractical to reproduce the actual production scenario so this fix is largely based on using existing known-to-work api methods.
